src/test/java/seedu/address/logic/parser/RsvpCommandParserTest.java

```diff
@@ -0,0 +1,52 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_EVENT_ID;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PERSON;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_RSVP_STATUS;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.RsvpCommand;
+import seedu.address.model.rsvp.RsvpStatus;
+
+public class RsvpCommandParserTest {
+
+    private static final String MESSAGE_INVALID_FORMAT =
+            String.format(MESSAGE_INVALID_COMMAND_FORMAT, RsvpCommand.MESSAGE_USAGE);
+
+    private RsvpCommandParser parser = new RsvpCommandParser();
+
+    @Test
```

Comment:
Likewise, for this test case for missing parts in the RsvpCommand, let's not have more than one missing input in each assertion.

Perhaps we could do it this way

```
// Missing PREFIX_EVENT_ID
assertParseFailure(parser, RsvpCommand.COMMAND_WORD + " "
        + PREFIX_PERSON + "1 " + PREFIX_RSVP_STATUS + "CC" ,  MESSAGE_INVALID_FORMAT);
// Missing PREFIX_PERSON
assertParseFailure(parser, RsvpCommand.COMMAND_WORD + " "
        + PREFIX_EVENT_ID + "1 " + PREFIX_RSVP_STATUS + "CC",  MESSAGE_INVALID_FORMAT);
// Missing PREFIX_RSVP_STATUS
assertParseFailure(parser, RsvpCommand.COMMAND_WORD + " "
        + PREFIX_EVENT_ID + "1 " + PREFIX_PERSON + "1 ", MESSAGE_INVALID_FORMAT);
```